volumic_arma_ultra.xml.fdm_material

```diff
@@ -0,0 +1,29 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material">
+  <metadata>
+    <name>
+      <brand>Volumic</brand>
+      <material>NYLON</material>

```

Comment:
Please write Nylon in normal case so that it matches with other materials with material type `Nylon`.
```suggestion
      <material>Nylon</material>
```
This helps for people who make customised profiles for Nylon materials.

Reply:
ok, change made.